datarobot_batch_scoring/batch_scoring.py

```diff
@@ -756,6 +757,10 @@ def run_batch_predictions(base_url, base_headers, user, pwd,
     first_row = first_row._replace(data=first_row_data)
     authorize(user, api_token, n_retry, endpoint, base_headers, first_row, ui)
 
+    print('_' * 80)
```

Comment:
Uhh... thanks for catching... my debug Output...
Am 06.06.2016 1:14 nachm. schrieb "Kyrylo Perevozchikov" <
notifications@github.com>:

> In datarobot_batch_scoring/batch_scoring.py
> <https://github.com/datarobot/batch-scoring/pull/28#discussion_r65873368>:
>
> > @@ -756,6 +757,10 @@ def run_batch_predictions(base_url, base_headers, user, pwd,
> >      first_row = first_row._replace(data=first_row_data)
> >      authorize(user, api_token, n_retry, endpoint, base_headers, first_row, ui)
> >
> > +    print('_' * 80)
>
> unnecessary
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/datarobot/batch-scoring/pull/28/files/47278abc3940dd5864ff912339df9c2e23dde60b#r65873368>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAG0cqb_U-Jg5HwMlRXfTXGpkfT9-h72ks5qJAEUgaJpZM4IuwVI>
> .
>